EugeneMunya/QuickCredit3
# #166183913 Viewing loans are not repaid and are repaid based on query parameters

#### What does this PR do?
Add functionality for retrieving loans that are paid and the ones are not repaid in the database. 
#### Description of Task to be completed?
At this point viewing, all loans that are not repaid and the ones are repaid is possible.
#### How should this be manually tested?
After cloning this repo and running npm start, then open postman and use a link looks like this localhost:3000/api/v1/repaid/loans?status=pending&repaid=false
 but make sure that you have logged in.
and pass your token as an authorization header.
#### What are the relevant pivotal tracker stories?
#166183913
#### Screenshots
![db](https://user-images.githubusercontent.com/36619897/58250581-05dac080-7d62-11e9-97ae-06f344f3d2c7.jpeg)

#### Questions
No questions
